spec/features/visitors/sign_up_spec.rb

```diff
@@ -209,12 +213,12 @@
     expect(current_url).to eq confirm_url
   end
 
-  scenario 'visitor confirms more than once' do
+  scenario 'visitor confirms more than once while signed in' do
```

Comment:
The way I've been using scenarios is to describe the context in the title, and read the expectations in the test to see what is expected to happen. For slow feature tests, it is recommended to combine all expectations in the same test, to avoid running the setup multiple times. For example, instead of something like this, which would run the `before` block for each `it` block:
```ruby
context 'visitor confirms more than once' do
  before do
    create(:user, :signed_up)
    sign_in_user
    send_confirmation_instructions
    click_link_in_email
  end

  it 'does not display flash message' do
    expect(page).to_not have_content('foo')
  end

  it 'redirects to home page' do
    expect(current_url).to eq root_url
  end

  etc...
end
```
You would do this:
```ruby
context 'visitor confirms more than once' do
  it 'redirects to root url without flash messages' do
    create(:user, :signed_up)
    sign_in_user
    send_confirmation_instructions
    click_link_in_email

    expect(page).to_not have_content('foo')
    expect(current_url).to eq root_url
  end
end
```
Sometimes, there might be more expectations than can fit in the `it` title, and I think that's OK. I don't think the `it` title should explain all the expectations.

In general, I prefer the `context` + `it` syntax because it allows you to specify both the scenario and the expectation, whereas with the `scenario` syntax, you have to either describe both in the scenario title, or add comments above the scenario.

Perhaps we should choose one style for consistency?